docs/docs/docs/01-core/content-releases/01-backend.md

```diff
@@ -153,3 +184,19 @@ Exposes validation functions to run before performing operations on a Release
 ```
 packages/core/content-releases/server/src/services/validation.ts
 ```
+
+## Migrations
+
+We have two migrations that we run every time we sync the content types.
+
+### `deleteActionsOnDisableDraftAndPublish`
+
+Every time an user disabled Draft and Publish in one Content Type we make sure to remove all the release actions related to entries of that content type to avoid errors.
```

Comment:
A user, it's an exception to the rule 🤦‍♂️ 
```suggestion
When a user disables Draft and Publish in one Content Type we make sure to remove all the release actions related to entries of that content type to avoid errors.
```

I think when disabling draft and publish all _drafts_ are deleted. This makes me think should probably not delete _all_ entries of that contentType but rather just the drafts 😬